Sources/Ink/Internal/List.swift

```diff
@@ -5,9 +5,13 @@
 */
 
 internal struct List: Fragment {
+    private static let orderedListMarkers: Set<Character> = [".", ")"]
+    
     var modifierTarget: Modifier.Target { .lists }
 
+    private var listMarker: Character
     private var isOrdered: Bool
+    private var startingIndex: Int?
```

Comment:
Since this data only makes sense if `isOrdered == true`, I think we should merge those two states into an enum:
```
extension List {
    enum Kind {
        case unordered
        case oredered(startingIndex: Int)
    }
}
```
Just to have a single source of truth, rather than two.